ark/segmentation/marker_quantification.py

```diff
@@ -5,16 +5,108 @@
 
 import xarray as xr
 
-from skimage.measure import regionprops_table
+from skimage.measure import moments, regionprops, regionprops_table
 
 from ark.utils import io_utils, load_utils, misc_utils, segmentation_utils
 from ark.segmentation.signal_extraction import extraction_function
 
 import ark.settings as settings
 
 
+def centroid_dif(prop):
+    """Return the cell centroid shift for a cell
+
+    Args:
+        prop (skimage.measure.regionprops):
+            The property information for a cell returned by regionprops
+
+    Returns:
+        float:
+            The centroid shift for the cell
+    """
+
+    cell_image = prop.image
+    cell_M = skimage.measure.moments(cell_image)
+    cell_centroid = cell_M[1, 0] / cell_M[0, 0], cell_M[0, 1] / cell_M[0, 0]
+
+    convex_image = prop.convex_image
+    convex_M = skimage.measure.moments(convex_image)
+    convex_centroid = convex_M[1, 0] / convex_M[0, 0], convex_M[0, 1] / convex_M[0, 0]
+
+    centroid_dist = np.sqrt((cell_centroid[1] - convex_centroid[1]) ** 2 +
+                            (cell_centroid[0] - convex_centroid[0]) ** 2)
+
+    centroid_dist /= np.sqrt(prop.area)
+
+    centroid_dif.append(centroid_dist)
+
+    return centroid_dif
+
+
+def num_concavities(prop):
```

Comment:
Yeah definitely

Reply:
cool cool.  then in terms of formatting the extra regionprops functions, they should probably take arbitrary kwargs, or use an if/else-if 'tree' instead of a global string->function map.  (I know we've had that philosophical discussion before with something else; I still stand on team function map)

Reply:
@ackagel yes, I agree that `kwargs` is the better way to go.

@ngreenwald the way `Segment_Image_Data.ipynb` is set up right now, it's not obvious to the user that they can pass in a different `extraction` parameter or a `kwargs` dictionary to tweak how `signal_extraction`/`regionprops_extraction` works. Is it time to add a sample `kwargs` dict in the notebook, say before the cell which calls `generate_cell_table`?

Reply:
Lets keep it on the secret menu for now. There's currently not a reason to use anything besides default extraction, so I'd rather keep things simple until we have a clear use-case for this functionality.